src/sentry/issues/endpoints/project_event_details.py

```diff
@@ -46,13 +51,15 @@ def wrap_event_response(
                 group_id=event.group_id,
                 environments=environments,
                 event=event,
+                conditions=conditions,
             )
         else:
-            conditions = []
+            legacy_conditions = []
             if environments:
-                conditions.append(["environment", "IN", environments])
+                legacy_conditions.append(["environment", "IN", environments])
+
             _filter = eventstore.Filter(
-                conditions=conditions,
+                conditions=legacy_conditions,
                 project_ids=[event.project_id],
                 group_ids=[event.group_id],
             )
```

Comment:
This is the codepath actually used in production, which will not be affected by these changes until we can roll out the `eventstore.adjacent_event_ids_use_snql` option.